test/customized_run_test.py

```diff
@@ -27,6 +27,7 @@
 
 
 class DummyTask(luigi.Task):
+    task_namespace = 'customized_run'
```

Comment:
why is this now required? Is this related to path setup between nose and pytest?

Reply:
I added a comment in a542dee